pkg/outputformat/outputformat.go

```diff
@@ -0,0 +1,51 @@
+// Package for having structured access to our output formats
+package outputformat
+
+import (
+	"fmt"
+	"slices"
+	"strings"
+)
+
+type OutputFormat string
```

Comment:
Add a stringer to this type, see https://github.com/editorconfig-checker/editorconfig-checker/pull/377#discussion_r1751019023

Reply:
done